### ISSUE TITLE
Doesn't compile with Swift 3.1

### ISSUE THREAD

**Issue by Jeehut** [ole/Ampere#13]
Using the current version (0.1) I get the following error when trying to build this project with Xcode 8.3 and Swift 3.1:

```
UnitAreaTests.swift:17:27: Binary operator '/' cannot be applied to operands of type 'Measurement<UnitArea>' and 'Measurement<UnitLength>'
```

I also wonder why the test classes build when I run the framework with Cmd+B ...

**Comment by ole** [comment]
Thanks for reporting this, I'll look into it.

> I also wonder why the test classes build when I run the framework with Cmd+B

That's intentional. I wanted to notice as soon as possible when tests fail to compile (e.g. when changing the API). I found this very useful during development.

**Comment by schmichri** [comment]
+1

**Comment by ole** [comment]
I pushed a fix (#16) to master. Will make a new release soon.

**Comment by ole** [comment]
I just published v0.2 with the fix.
